airflow/www/templates/airflow/dag.html

```diff
@@ -151,7 +151,8 @@ <h4 class="pull-right js-dataset-triggered" style="user-select: none;-moz-user-s
         {% endif %}
         {% if dag_model is defined and dag_model.next_dagrun is defined and dag_model.schedule_interval != 'Dataset' %}
           <p class="label label-default js-tooltip" style="margin-left: 5px" id="next-run" data-html="true" data-placement="bottom">
-            Next Run ID: <time datetime="{{ dag_model.next_dagrun }}">{{ dag_model.next_dagrun }}</time>
+            Next Run ID: <time datetime="{{ dag_model.next_dagrun }}">{{ dag_model.next_dagrun }}
+              {{ dag_model.next_dagrun.tzinfo }}</time>
```

Comment:
@bbovenzi @pierrejeambrun With the fix added in the dag.html, I could see that time has timezone info (both in text UTC and numerical +00:00 representation ) on loading, but then after loading it goes back to the old state. Am I missing something to notice?
<img width="1041" alt="Screenshot 2024-03-26 at 12 07 41 PM" src="https://github.com/apache/airflow/assets/10162465/cc7d4882-c6f6-4c99-8758-68869eec6217">

Reply:
The above image is during load, and I am attaching below the image which shows up after the page got loaded

<img width="1041" alt="Screenshot 2024-03-26 at 12 15 08 PM" src="https://github.com/apache/airflow/assets/10162465/ddab1af1-adc7-4e48-84a0-a59dfd39d950">

Reply:
The only change needed is to add `data-with-tz="true"` to `<time>`

Reply:
I see @bbovenzi 
I didn't know such an attribute exists for time tag( googling also didn't help). Thank you so much. I have a long way to go in exploring the frontend side.